lang/lua-eco/Makefile

```diff
@@ -65,6 +65,7 @@ Package/lua-eco-ip=$(call Package/lua-eco/Module,ip utils,+lua-eco-netlink)
 Package/lua-eco-nl80211=$(call Package/lua-eco/Module,nl80211,+lua-eco-netlink)
 Package/lua-eco-ssh=$(call Package/lua-eco/Module,ssh,+lua-eco-socket +libssh2)
 Package/lua-eco-packet=$(call Package/lua-eco/Module,packet,+lua-eco-nl80211)
+Package/lua-eco-uci=$(call Package/lua-eco/Module,uci,+libuci)
```

Comment:
How this is related to update 3.10.0? This does not look right as it is not described anywhere. Your commit looks like this one, which was done in past https://github.com/openwrt/packages/commit/733425e1da8b4193663938bc57f9554dc014e03b , but actually it does more changes than necessary. Could you please look at it more? :)